packages/zowe-explorer/src/trees/shared/SharedUtils.ts

```diff
@@ -23,6 +23,12 @@ import { SharedContext } from "./SharedContext";
 import { Definitions } from "../../configuration/Definitions";
 
 export class SharedUtils {
+    public static async copyExternalLink(this: void, node: IZoweTreeNode): Promise<void> {
+        if (node?.resourceUri != null) {
+            await vscode.env.clipboard.writeText(`vscode://Zowe.vscode-extension-for-zowe?${node.resourceUri.toString()}`);
```

Comment:
Not saying that this will ever change, but curious if we should use some constant values here 😅

Reply:
As you mentioned, I doubt we'll ever change our extension ID, but I updated the logic to use the extension context's `extension.id`.

Resolved in 51c1699